.ci_support/build_all.py

```diff
@@ -207,9 +240,47 @@ def build_folders(recipes_dir, folders, arch, channel_urls):
         conda_build.api.build([recipe], config=get_config(arch, channel_urls))
 
 
+def build_folders_rattler_build(recipes_dir: str, platform, arch, channel_urls: list[str]):
+    config = get_config(arch, channel_urls)
+
+    # Remove the example recipes to ensure that they are not also build.
+    for example_recipe in EXAMPLE_RECIPE_FOLDERS:
+        rmtree(os.path.join(recipes_dir, example_recipe), ignore_errors=True)
```

Comment:
This would become a problem locally if the user accidentally commits after debugging something, but I think we copy the staged-recipes folder before we start the script, so we are ok.